docs/getting-started/guides/how-to-understand-strategies-descriptions.md

```diff
@@ -0,0 +1,111 @@
+- Yearn Vault Token (yvToken) is an ERC20-compatible token that represents a [position](#position).
+- For example: yvDAI represents a position in the DAI Yearn Vault.
+### Strategy
+- A [smart contract↗](https://ethereum.org/en/developers/docs/smart-contracts/) that knows how to allocate funds into a specific [yield](#yield--profit)-generating strategy.
+### Strategist
+- A person that creates and maintains one or more strategies.
+
+# Finance
+
+### Yield / Profit
+- Money (or tokens) made by an investment that surpasses the invested initial amount.
+### Interest
+- Fee charged for [lending](#lend--borrow-against) money.
+### Compound / Autocompound
+- Compound is when you deposit yields into the same investment that it came from.
+- Autocoumpound is when a tool does compounding automatically for you.
+- There is also a protocol called Compound Finance (COMP) but that is different.
+### Position
+- Total amount of money (or tokens) allocated into a investment.
+### Hedge
+- A position in a contrarian investment to one you already have, so if things go wrong in one thesis you offset losses by gains in the other position. 
+### Arbitrage
+- Buy and Sell tokens for instant [profit](#yield--profit). For example: Eth is 2000$ at one place and 1950$ at another, so you quickly buy from the cheaper source and sell on the more expensive source for 50$ profit.  
+### Vest
+- Gradual [unlocking](#unlock) of tokens or stocks.
+- For example: if you receive 4000 tokens in a 4 year vesting it usually means you get 1000 tokens/year (unlocked on a daily or monthly basis depending on the contract).
+### Collateral
+- An amount of money (or tokens) that gets [locked](#lock--unlock) until a pre-determined agreement is met, commonly at risk of [liquidation](#liquidation)
+### Liquidation
+- When you fail to repay a [loan](#lend--borrow-against) or other agreement which involves [collateral](#collateral) and the collateral is sent to other parties to repay and make them whole, so you can't access the collateral anymore.
+- For example: you add 100 Eth as collateral and borrow 75% of it's value as dollars, but you fail to repay the loan in due time, so your collateral gets *liquidated* which costs you 25% of the initial amount deposited.
+
+### Lend / Borrow Against
+- Lending is when you provide liquidity to someone else for some time and expect them to pay you back in the future.
+- Borrowing is when you take money from a lender.
+- Borrowing *against* is used to specify the collateral used by the borrower. Example: "I borrowed 5000$ against my car, so if I don't repay the loan the bank can take it".
+- In DeFi this operation is always collateralized since there are no resources to charge debt.
+
+# DeFi
+
+### Yield Aggregator / Yield Optimizer
+- DeFi protocols like Yearn that provides a single interface that aggregates/optimizes users deposits in a single place and reallocates them in an optimal way.
```

Comment:
I would avoid using the same word that you are defining. It seems that an aggregator aggregates and an optimizer optimizes ;)

You could say something along the lines of: `allocating capital in different yield sources, leveraging autocompounding to improve the yield`